pkg/report/testdata/linux/decompile/arm64/0.out

```diff
@@ -46,4 +46,4 @@ Code disassembly (best guess):
    4:	d34cfc00 	lsr	x0, x0, #12
    8:	97fcf6fe 	bl	0xfffffffffff3dc00
    c:	35fffc20 	cbnz	w0, 0xffffffffffffff90
-  10:	d4210000 	brk	#0x800 <-- trapping instruction
+* 10:	d4210000 	brk	#0x800 <-- trapping instruction
```

Comment:
This is arm's trap instruction.
Should it also be skipped?

Reply:
Hmm, yes, that should've also been skipped. Added this instruction to the `linuxSkipTrapInstrRe`.